resolver.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"skillfactory/SF-34_6_1/calc"
+	"strconv"
+)
+
+func Resolver(l []string) ([]string, error) {
+	if len(l) == 0 {
+		return nil, errors.New("на входе функции пустой список")
+	}
+	result := []string{}
+
+	re := regexp.MustCompile(`(\d+)([+-\/\*])(\d+)[=]([\?])`)
+	calc := calc.NewCalculator()
+
+	for _, s := range l {
+		groups := re.FindStringSubmatch(s)
+		if len(groups) != 5 {
+			continue
+		}
+		n1, err := strconv.ParseFloat(groups[1], 64)
+		if err != nil {
+			continue
+		}
+		op := groups[2]
+		n2, err := strconv.ParseFloat(groups[3], 64)
+		if err != nil {
+			continue
+		}
+		res, err := calc.Calculate(n1, op, n2)
+		if err != nil {
+			continue
+		}
```

Comment:
не забывайте про отступы, очень влияет на чтение кода
```suggestion
		groups := re.FindStringSubmatch(s)
		if len(groups) != 5 {
			continue
		}
		
		n1, err := strconv.ParseFloat(groups[1], 64)
		if err != nil {
			continue
		}
		
		op := groups[2]
		n2, err := strconv.ParseFloat(groups[3], 64)
		if err != nil {
			continue
		}
		
		res, err := calc.Calculate(n1, op, n2)
		if err != nil {
			continue
		}
```